en-eng/Zigate-Plugin-FAQ.md

```diff
@@ -11,4 +11,12 @@ This is more an OS issue, but here are few hints
 1. Start the Plugin
 1. Trigger the pairing process from the device
 
+## I don't get the device updated in Domoticz when interacting via a remote control or manualy
+
+If I power Off an Ikea Bulb (with a physical switch), when switching it on I don't have a status updated in Domoticz. I have the same issue if using a remote controller .
```

Comment:
_For example :_ If you power off an Ikea bulb (with the physical switch) and then switch it on again the status of the bulb is not updated in Domoticz. and are having the same issue with a remote controller which also doesn't update it's current state to Domoticz.

If you encounter problems like the above example, it is most likely the automatic reporting is not configured correctly from the device end.  Please follow below steps to fix this issue: